drivers/pinctrl/intel/pinctrl-intel.c

```diff
@@ -1503,6 +1546,12 @@ int intel_pinctrl_suspend(struct device *dev)
 		base = community->regs + community->ie_offset;
 		for (gpp = 0; gpp < community->ngpps; gpp++)
 			communities[i].intmask[gpp] = readl(base + gpp * 4);
+
+		if (pctrl->quirks | PINCTRL_QUIRK_KEEP_HOSTOWN) {
```

Comment:
`(x | 1)` is always true. I believe you wanted
```suggestion
		if (pctrl->quirks & PINCTRL_QUIRK_KEEP_HOSTOWN) {
```